ViewModels/LoginViewModel.cs

```diff
@@ -51,32 +49,42 @@ public LoginViewModel(INavigation navigation)
         _navigation = navigation;
         LoginCommand = new Command(OnLogin);
         CreateAccountCommand = new Command(OnCreateAccount);
+        
+        _backendService = new BackendService(ClientSingleton.GetInstance("http://api.combatcritters.ca:4000"));
     }
 
     private async void OnLogin()
     {
-        // var result = await BackendService.LoginAsync(new UserCredentials
-        // {
-        //     Username = Username,
-        //     Password = Password
-        // });
-
-        //For TEST
-        var result = true;
-        if (result)
+        try
         {
-            // Navigate to ProfilePage through Shell after successful login
-            (Application.Current as App)?.NavigateToAppShell();
+            var loginRequestToBackend = await _backendService.LoginAsync(new UserCredentials{
+                Username = Username,
+                Password = Password
+            });
+            
+            if (loginRequestToBackend)
+            {
+                //The client con has been stored in backendservices
+
+                //Navigate to User Profile page
+                (Application.Current as App)?.NavigateToAppShell();
+            }
+            else
+            {
+                
+                //show an error message
+                //Maybe display a notification in UI
```

Comment:
todo should be made into an issue / removed

Reply:
Okay, issued created